.github/workflows/python-tests.yml

```diff
@@ -31,12 +33,6 @@ jobs:
         # - windows-2019
         # - windows-2016
         include:
-        # Pre-release versions (GH-shipped)
-        - os: ubuntu-20.04
-          python-version: 3.9.0-beta.4 - 3.9.0
-        # Pre-release versions (deadsnakes)
-        - os: ubuntu-20.04
-          python-version: 3.9-beta
```

Comment:
It's funny that I was just about to submit exactly this change right before seeing your PR. So I ended up sending only the metadata update: https://github.com/pypa/setuptools/pull/2421.